global_setup.sh

```diff
@@ -65,6 +76,24 @@ if [[ -z $prompt ]] || [[ $prompt == "Y" ]]; then
     $(echo "" >> ~/.bashrc) 
 fi;
 
+say "[STEP 5] Add git templatedir folder"
+$(git config --global --replace-all init.templatedir '~/.git-templates')
+hook_path=~/.git-templates/hooks
+mkdir -p $hook_path
+
+write_hook $hook_path "applypatch-msg"
```

Comment:
MEJORA: Podemos meter este conjunto de mensajes en un array y luego iterar en un for